trips.js

```diff
@@ -0,0 +1,91 @@
+$(document).ready( function() {
```

Comment:
places like this where you're not using `this` make good places to use an arrow function.